src/util/PortableUtil.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/** @ignore *//** */
+
+import {DataInput, DataOutput} from '../serialization';
+import {
+    LocalDate,
+    LocalDateTime,
+    OffsetDateTime,
+} from '../core';
+import {IOUtil} from './IOUtil';
+
+/** @internal */
+export class PortableUtil {
+
+    static readLocalDateForPortable(inp: DataInput): LocalDate {
```

Comment:
I think we might omit the `ForPortable` name, as the class name suggests that it is for Portable